### PR TITLE
ISY-305: implement TokenUtil for reading claims from the bearer token

### PR DESCRIPTION
**Pull request by tbribram**

### CODE REVIEW

File: isy-security/src/main/java/de/bund/bva/isyfact/security/core/Berechtigungsmanager.java [hunk 1]
```diff
@@ -46,7 +46,9 @@ public interface Berechtigungsmanager {
      *
      * @param key the key to retrieve given attribute
      * @return the attribute in the access token for the given key, or {@code null}
+     * @deprecated this method is transferred to IsySecurityTokenUtil and will be removed from this interface in a future release
```
Comment (huebnercl) [inline isy-security/src/main/java/de/bund/bva/isyfact/security/core/Berechtigungsmanager.java]: Is it possible to replace the name `IsySecurityTokenUtil` by a link to this new class?

File: isy-security/src/main/java/de/bund/bva/isyfact/security/oauth2/util/IsySecurityTokenUtil.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+     */
+    private static final String PREFIX = "isy.security.oauth2.claim.";
+
+    /**
+     * Returns the value of the configuration property with the given suffix.
+     *
+     * @param suffix the suffix of the configuration property
+     * @return the String value of the configuration property
+     */
+    private static String getString(String suffix) {
```
Comment (huebnercl) [inline isy-security/src/main/java/de/bund/bva/isyfact/security/oauth2/util/IsySecurityTokenUtil.java]:
```suggestion
    private static String getConfigPropertyValueAsString(String suffix) {
```

please use this more expressive method name to recognize and follow more efficiently what's going on.

**Review by huebnercl — CHANGES_REQUESTED** [review]

File: isy-security/src/main/java/de/bund/bva/isyfact/security/oauth2/util/IsySecurityTokenUtil.java (hunk 2 above)
Comment (tbribram) [reply isy-security/src/main/java/de/bund/bva/isyfact/security/oauth2/util/IsySecurityTokenUtil.java]: Done: https://github.com/IsyFact/isyfact-standards/pull/111/commits/b9c7b551dbd79f2ff590ecc31b8e61c2d4cd380c

File: isy-security/src/main/java/de/bund/bva/isyfact/security/core/Berechtigungsmanager.java (hunk 1 above)
Comment (tbribram) [reply isy-security/src/main/java/de/bund/bva/isyfact/security/core/Berechtigungsmanager.java]: Done: https://github.com/IsyFact/isyfact-standards/pull/111/commits/b9c7b551dbd79f2ff590ecc31b8e61c2d4cd380c

**Review by huebnercl — APPROVED** [review]

**Review by huebnercl — DISMISSED** [review]

**Review by huebnercl — DISMISSED** [review]

**Review by huebnercl — APPROVED** [review]

**Pull request merged**